openexr-rs/src/version.rs

```diff
@@ -0,0 +1,273 @@
+use openexr_sys as sys;
+// use std::ops::{BitAnd, BitAndAssign, BitOr, BitOrAssign, BitXor, BitXorAssign};
+use std::os::raw::{c_char, c_int};
+
+bitflags::bitflags! {
+    /// The version flags represents the options that are supported in the library and the EXR file.
+    ///
+    pub struct VersionFlags: i32 {
+        /// File is tiled
+        const TILED_FLAG = 0x00000200;
```

Comment:
Now that it's scoped we should remove _FLAG from the flag names so it's `VersionFlags::TILED` rather than `VersionFlags::TILED_FLAG`